devtools-ci-index.yaml

```diff
@@ -2942,6 +2942,90 @@
                           vault-key: '{service_account_client_secret_cred}'
     <<: *contract_tests_provider_template
 
+- job-template: 
+    name: '{ci_project}-{git_repo}-pr-check-rh-che'  
+    scm:  
+        - git-scm:  
+            git_url: https://github.com/openshiftio/{git_repo}.git
+            git_basedir: '{git_repo}' 
+    beforeScript: | 
+        # if the PR to saas-openshiftio is done immediatelly after the commit 
+        # to dependency's master, it might be still deploying to prod-preview 
+        echo "Sleeping for 1 min" 
+        sleep 60
+    triggers:  
+        - github-pull-request:  
+            github_hooks: '{github_hooks}'  
+            status-context: 'ci.centos.org rh-che test against prod-preview'  
+            failure-comment: "**The rh-che test against prod-preview failed.**\n\n  
+            | Link | URL |\n  
+            | ---- | :-: |\n  
+            | job log | https://ci.centos.org/job/$JOB_NAME/$BUILD_NUMBER/console |\n 
+            | artifacts | http://artifacts.ci.centos.org/devtools/saas-openshiftio/$JOB_NAME/$BUILD_NUMBER |\n\n 
+            **Please review the job results and re-run the job by commenting [test] if necessary.**"  
+            error-comment: "**The rh-che test against prod-preview failed.**\n\n  
+            | Link | URL |\n  
+            | ---- | :-: |\n  
+            | job log | https://ci.centos.org/job/$JOB_NAME/$BUILD_NUMBER/console |\n 
+            | artifacts | http://artifacts.ci.centos.org/devtools/saas-openshiftio/$JOB_NAME/$BUILD_NUMBER |\n\n 
+            **Please review the job results and re-run the job by commenting [test] if necessary.**"  
+            <<: *github_pull_request_defaults 
+    properties:
+        - github:
+            url: https://github.com/openshiftio/{git_repo}
+    wrappers:
+        - vault-secrets:
+            <<: *vault_defaults
+            secrets:
+                - secret-path: 'devtools-osio-ci/rhche-saas-prcheck'
+                  secret-values:
+                    - env-var: 'PASSWORD'
+                      vault-key: 'password'
+                    - env-var: 'USERNAME'
+                      valut-key: 'username'
+                    - env-var: 'EMAIL'
+                      valut-key: email
+    builders:
+        - shell: |
+            {beforeScript}
+        
+            echo "Installing docker..."
+            yum install -y docker | cat # Suppress multiple-line output for package installation
```

Comment:
this is not going to work because you can't install / run custom things right on ci slave. 
1) I would recommend to move all build logic in build script like `pr-check.sh` that lives in your repo
2) you should execute that `pr-check.sh` script on duffy node via ssh but not on the CI slave since you don't have permissions there.  ex: https://github.com/openshiftio/openshiftio-cico-jobs/blob/master/devtools-ci-index.yaml#L531
3) to make it simpler you can create new job-template that will extend [job_template_defaults](https://github.com/openshiftio/openshiftio-cico-jobs/blob/master/devtools-ci-index.yaml#L468) which will provide all duffy logic to your cutom template and then you simply add things to it.

Reply:
@Katka92 fyi^

Reply:
@ibuziuk I've noticed that, but thank you for pinging me. I'm working on that. :)

Reply:
@riuvshin Hi so I tried to fix that. Could you please have a look?